projects/pdf-player-wc/src/app/app.module.ts

```diff
@@ -20,8 +20,7 @@ import { SunbirdPdfPlayerComponent } from '../../../sunbird-pdf-player/src/lib/s
     HttpClientModule,
     SunbirdPlayerSdkModule
   ],
-  providers: [],
-  entryComponents: [SunbirdPdfPlayerComponent]
+  providers: []
```

Comment:
Why entry component is removed

Reply:
While upgrding the version from 9 to 10, 
they mentioned:=> ```In your application projects, you can remove entryComponents NgModules and any uses of ANALYZE_FOR_ENTRY_COMPONENTS. They are no longer required with the Ivy compiler and runtime.```
 we can remove it, if we want. For now, i keep this as it is and tested both scenarios.